KNOWN_ISSUES.md

```diff
@@ -127,7 +128,58 @@ kubectl get --raw /api/v1/nodes/$NODE_NAME/proxy/metrics/resource
   **Workaround**
   
   Upgrade Docker to v19.03
-  
+
+## Hpa is unable to compute the replica count
+
+**Symptoms**
+
+Created a deployment and an HPA. when deployed the metrics server and the HPA is throwing errors about requests not defined on the deployment when clearly the resources are requested and limited
+
+**Debugging**
+
+* Make sure `kubectl top node` and `kubectl top pod` results are normal.
```

Comment:
users might not understand what normal means for  `kubectl top` please give example.